src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileRetriever.java

```diff
@@ -62,6 +63,8 @@ public UserProfile getRolesFromIdam(UserProfile userProfile, boolean isMultiUser
                 persistAudit(idamRolesInfo, userProfile);
                 // for multiple users get request , do not throw exception and continue flow
                 if (!isMultiUserGet) {
+                    UserProfileUtil.idam5xxxErrorResponse(idamRolesInfo.getStatusMessage(),
```

Comment:
can we rename this to `idam5xxErrorResponse` (remove one `x`)

Reply:
done